compiler/z/codegen/UnaryEvaluator.cpp

```diff
@@ -496,11 +496,6 @@ OMR::Z::TreeEvaluator::l2aEvaluator(TR::Node * node, TR::CodeGenerator * cg)
 
    node->setRegister(source);
 
-
-   if (firstChild->containsCompressionSequence())
```

Comment:
Could you please clarify if this is a functional fix ? i.e. what can go wrong if we decrement the reference count of the shift left tree that is not evaluated ?